packaging/docker/Dockerfile

```diff
@@ -14,46 +14,46 @@ ARG GF_UID="472"
 ARG GF_GID="472"
 
 ENV PATH=/usr/share/grafana/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
-    GF_PATHS_CONFIG="/etc/grafana/grafana.ini" \
-    GF_PATHS_DATA="/var/lib/grafana" \
-    GF_PATHS_HOME="/usr/share/grafana" \
-    GF_PATHS_LOGS="/var/log/grafana" \
-    GF_PATHS_PLUGINS="/var/lib/grafana/plugins" \
-    GF_PATHS_PROVISIONING="/etc/grafana/provisioning"
+  GF_PATHS_CONFIG="/etc/grafana/grafana.ini" \
+  GF_PATHS_DATA="/var/lib/grafana" \
+  GF_PATHS_HOME="/usr/share/grafana" \
+  GF_PATHS_LOGS="/var/log/grafana" \
+  GF_PATHS_PLUGINS="/var/lib/grafana/plugins" \
+  GF_PATHS_PROVISIONING="/etc/grafana/provisioning"
 
 WORKDIR $GF_PATHS_HOME
 
 RUN apk add --no-cache ca-certificates bash && \
-    apk add --no-cache --upgrade --repository=http://dl-cdn.alpinelinux.org/alpine/edge/main openssl musl-utils
+  apk add --no-cache --upgrade --repository=http://dl-cdn.alpinelinux.org/alpine/edge/main openssl musl-utils
 
 # PhantomJS
 RUN if [ `arch` = "x86_64" ]; then \
-      apk add --no-cache --virtual phantomjs-utils curl && \
-      cd /tmp && \
-      curl -Ls https://github.com/dustinblackman/phantomized/releases/download/2.1.1/dockerized-phantomjs.tar.gz | tar xz && \
-      cp -R lib lib64 / && \
-      cp -R usr/lib/x86_64-linux-gnu /usr/lib && \
-      cp -R usr/share/fonts /usr/share && \
-      cp -R etc/fonts /etc && \
-      rm -rf /tmp/* && \
-      apk del --no-cache phantomjs-utils; \
-    fi
+  apk add --no-cache --virtual phantomjs-utils curl && \
+  cd /tmp && \
+  curl -Ls https://github.com/dustinblackman/phantomized/releases/download/2.1.1/dockerized-phantomjs.tar.gz | tar xz && \
+  cp -R lib lib64 / && \
+  cp -R usr/lib/x86_64-linux-gnu /usr/lib && \
+  cp -R usr/share/fonts /usr/share && \
+  cp -R etc/fonts /etc && \
+  rm -rf /tmp/* && \
+  apk del --no-cache phantomjs-utils; \
+  fi
 
 COPY --from=0 /tmp/grafana "$GF_PATHS_HOME"
 
 RUN mkdir -p "$GF_PATHS_HOME/.aws" && \
-    addgroup -S -g $GF_GID grafana && \
-    adduser -S -u $GF_UID -G grafana grafana && \
-    mkdir -p "$GF_PATHS_PROVISIONING/datasources" \
```

Comment:
I think the removal of custom indentation for the mkdir paths makes these lines a bit harder to read. Not sure if this is intentional or just the result of a reformat of this file, but I would consider keeping some of the custom formattings for readability.

Reply:
@xlson, possibly this was done automatically by VS Code, I can't seem to turn this "feature" off for Dockerfiles 😔